VirtoCommerce.DynamicExpressionsModule.Data/Promotion/Helpers/PromotionEvaluationContextExtension.cs

```diff
@@ -71,11 +72,37 @@ public static bool IsAnyLineItemExtendedTotal(this PromotionEvaluationContext co
                     .Any();
         }
 
+        public static bool IsAnyLineItemExtendedTotalNew(this PromotionEvaluationContext context, decimal lineItemTotal, decimal lineItemTotalSecond, string compareCondition, string[] excludingCategoryIds, string[] excludingProductIds)
```

Comment:
I think we can use override version of this function `IsItemsInStockQuantity`  only with new parameters.
And call it in the old version of this function